### DISCUSSION TITLE
Set `TERM_PROGRAM`

### DISCUSSION THREAD

**Discussion by Qubik65536** [migueldeicaza/SwiftTerm#317] [Q&A] [answered]
Hi, I'm trying to create some script for CodeEdit project that would only works if the terminal is running under CodeEdit app, so I thought about using `TERM_PROGRAM`. But nothing have been shown when I execute `echo $TERM_PROGRAM` (it's the same using the [TerminalApp](https://github.com/migueldeicaza/SwiftTerm/tree/main/TerminalApp)). Is there a way for us to set the `TERM_PROGRAM` while creating a new terminal?

**Comment by migueldeicaza** [comment, marked as answer]
The `startProcess` method in `LocalProcess` takes an optional `environment` array that contains the values you want to pass to the child process.
